client/me/purchases/manage-purchase/notices.jsx

```diff
@@ -195,8 +195,10 @@ class PurchaseNotice extends Component {
 		}
 
 		if (
-			! hasPaymentMethod( purchase ) &&
-			( ! canExplicitRenew( purchase ) || shouldAddPaymentSourceInsteadOfRenewingNow( purchase ) )
+			( ! hasPaymentMethod( purchase ) &&
+				( ! canExplicitRenew( purchase ) ||
+					shouldAddPaymentSourceInsteadOfRenewingNow( purchase ) ) ) ||
+			isPaidWithCredits( purchase )
```

Comment:
Although the current behavior is definitely wrong, I don't think we want this change since it introduces the same problem as in https://github.com/Automattic/wp-calypso/pull/46379#issuecomment-707777428 (which is worse).  Probably better to leave the fix for this problem to that issue and just keep the current logic in place here.

Reply:
Cool, sounds good. I'll separate this out in the new year so we can ship the text changes.

Reply:
I went ahead and reverted this part so I can deploy this.